code/modules/games/cards.dm

```diff
@@ -267,16 +283,24 @@
 	icon = 'icons/obj/playing_cards.dmi'
 	icon_state = "empty"
 	w_class = WEIGHT_CLASS_TINY
-	throw_speed = 4
-	throw_range = 20
-	throwforce = 0
-	force = 0
 	var/maxcardlen = 20
 	actions_types = list(/datum/action/item_action/remove_card, /datum/action/item_action/discard)
 
 	var/concealed = FALSE
 	var/list/cards = list()
 	var/parentdeck = null
+	var/pickedcard = null
+
+/obj/item/cardhand/proc/update_values()
+	if(parentdeck)
+		var/obj/item/deck/D = parentdeck
+		hitsound = D.card_hitsound
+		force = D.card_force
+		throwforce = D.card_throwforce
+		throw_speed = D.card_throw_speed
+		throw_range = D.card_throw_range
+		attack_verb = D.card_attack_verb
+		resistance_flags = D.card_resistance_flags
```

Comment:
Might be easier at this point to just make a new standard card from the deck and assign the new card's suit